fdm-app/app/routes/farm.$b_id_farm.atlas.elevation.tsx

```diff
@@ -0,0 +1,75 @@
+import { AtlasFields } from "@/components/custom/atlas-fields"
+import { getMapboxToken } from "@/components/custom/atlas/atlas-mapbox"
+import { Button } from "@/components/ui/button"
+import { fdm } from "@/lib/fdm.server"
+import { getFields } from "@svenvw/fdm-core"
+import {
+    type LoaderFunctionArgs,
+    NavLink,
+    data,
+    useLoaderData,
+} from "react-router"
+import wkx from "wkx"
+
+export async function loader({ request, params }: LoaderFunctionArgs) {
+    // Get the farm id
```

Comment:
you only show properties of the selected farm fields, ins't? A future feature might be to show (open) properties on regional level in the region around the users farm (for a given zoom level only).

Reply:
Yes, the viewstate is based on the fields of the selected farm. Other layers can be added in the future in combination with the fields